tests/nativeapp/codegen/test_compiler.py

```diff
@@ -27,7 +27,7 @@
 
 @pytest.fixture()
 def test_proj_def():
-    return ProjectDefinition(
```

Comment:
Can we mess around with the export so we can keep `ProjectDefinition(...)` everywhere even though it's now different based on the feature flag?